pages/linux/olevba.md

```diff
@@ -0,0 +1,29 @@
+# olevba
+
+> Parse OLE and OpenXML files (e.g., DOC, XLS, PPT, etc.) to extract VBA macros, deobfuscate, and analyze malicious code.
+> Part of the `python-oletools` suite.
+> For more information: <http://decalage.info/python/oletools>.
```

Comment:
The "more information" link is just a blank page for me. Not sure if this is temporary or if it's been broken for a while. We can link to the GitHub page if needed
```suggestion
> For more information: <https://github.com/decalage2/oletools>.
```